TicTacToe/Board.cs

```diff
@@ -0,0 +1,39 @@
+namespace TicTacToe
+{
+    public class Board
+    {
+        private int[,] _boardArray;
+        
+        public Board()
+        {
+            _boardArray = new[,]
+            {
+                {0, 0, 0},
+                {0, 0, 0},
+                {0, 0, 0}
+            };
+        }
+
+        public int[,] GetBoard()
+        {
+            return _boardArray;
+        }
+
+        public void PlaceMarker(Move move, int player)
+        {
+            var (coordX, coordY) = move.GetCoord();
+            _boardArray[coordX, coordY] = player;
```

Comment:
I think an improvement here would be to change player to playerToken or playerNumber, it would be a bit close to showing what it actually is rather than a player.